src/main/resources/docs/description/description.json

```diff
@@ -299,32 +299,57 @@
   },
   {
     "patternId": "rulesets-design.xml-CouplingBetweenObjects",
+    "parameters": [
+      {
+        "name": "maximum",
+        "default": 13,
+        "description": "Maximum number of acceptable dependencies."
+      }
+    ],
     "title": "Prohibit of coupling between objects",
-    "description": "Boolean methods should be called 'isX()' or 'hasX()'",
+    "description": "A class needs to have low coupling",
     "timeToFix": 5
   },
   {
-    "patternId": "rulesets-naming.xml-BooleanGetMethodName",
-    "title": "Enforce boolean naming convention",
-    "description": "Boolean methods should be called 'isX()' or 'hasX()'",
-    "timeToFix": 5
-  },
-  {
-    "patternId": "rulesets-naming.xml-BooleanGetMethodName",
-    "title": "Enforce boolean naming convention",
-    "description": "Boolean methods should be called 'isX()' or 'hasX()'",
+    "patternId": "rulesets-design.xml-DevelopmentCodeFragment",
+    "parameters": [
+      {
+        "name": "unwanted-functions",
+        "default": "var_dump,print_r,debug_zval_dump,debug_print_backtrace",
+        "description": "Comma separated list of suspect function images."
+      },
+      {
+        "name": "ignore-namespaces",
+        "default": false,
+        "description": "Ignore namespaces when looking for dev. fragments"
+      }
+    ],
+    "title": "Prohibit calling debug function which is mostly used during development.",
+    "description": "Functions like var_dump(), print_r() etc. are normally only used during development and therefore such calls in production code are a good indicator that they were just forgotten.",
     "timeToFix": 5
   },
   {
-    "patternId": "rulesets-naming.xml-BooleanGetMethodName",
-    "title": "Enforce boolean naming convention",
-    "description": "Boolean methods should be called 'isX()' or 'hasX()'",
+    "patternId": "rulesets-cleancode.xml-BooleanArgumentFlag",
+    "title": "A boolean flag argument is a reliable indicator for a violation of the Single Responsibility Principle (SRP)",
+    "description": "You can fix this problem by extracting the logic in the boolean flag into its own class or method",
     "timeToFix": 5
   },
   {
-    "patternId": "rulesets-naming.xml-BooleanGetMethodName",
-    "title": "Enforce boolean naming convention",
-    "description": "Boolean methods should be called 'isX()' or 'hasX()'",
+    "patternId": "rulesets-codesize.xml-TooManyPublicMethods",
+    "title": "Excessive public methods",
+    "description": "This pattern reports classes with a high number of public methods.",
+    "parameters": [
+      {
+        "name": "maxmethods",
+        "default": 10,
+        "description": "The method count reporting threshold"
+      },
+      {
+        "name": "ignorepattern",
+        "default": "(^(set|get))i",
```

Comment:
I don't think `description.json` should contain the `default`s.
Check the format [here](https://github.com/codacy/codacy-example-tool#description-structure)